### PR TITLE
Setup translation infrastructure

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

**Comment by nadjaheitmann** [comment]
Oh, I am sorry I saw this... Just over a year after you've opened the PR :see_no_evil: I think we should get it in, though. Do you mind rebasing?

**Comment by ofedoren** [comment]
I forgot about this myself :) Rebased.

**Comment by nadjaheitmann** [comment]
Rubocop has some minor complaints. Too bad, I just merged theforeman-rubocop into the plugin this morning ;)

**Comment by ofedoren** [comment]
Should be green now :)

File: locale/hammer_cli_foreman_scc_manager.pot [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the hammer_cli_foreman_scc_manager package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: hammer_cli_foreman_scc_manager 0.1.0\n"
```
Comment (nadjaheitmann) [inline locale/hammer_cli_foreman_scc_manager.pot:9]: Shouldn't this be 0.2.1 ?

File: locale/en/hammer_cli_foreman_scc_manager.po [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# English translations for hammer_cli_foreman_scc_manager package.
+# Copyright (C) 2023 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the hammer_cli_foreman_scc_manager package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: hammer_cli_foreman_scc_manager 0.1.0\n"
```
Comment (nadjaheitmann) [inline locale/en/hammer_cli_foreman_scc_manager.po:8]: Same here about the version number. It is 0.2.1 and I want to bump a new version today such that I can update the packaging.

File: .tx/config [hunk 3]
```diff
@@ -0,0 +1,9 @@
+[main]
+host = https://app.transifex.com
+
+[o:foreman:p:foreman:r:hammer_cli_foreman_scc_manager]
+file_filter = locale/<lang>/hammer_cli_foreman_scc_manager.edit.po
+source_file = locale/hammer_cli_foreman_scc_manager.pot
+source_lang = en
+type = PO
+resource_name = hammer_cli_foreman_scc_manager
```
Comment (nadjaheitmann) [inline .tx/config:9]: Do you actually submit that config file? I haven't seen it in any other projects.

Comment (nadjaheitmann) [reply .tx/config:9]: My bad, I did not search well enough :)

File: locale/en/hammer_cli_foreman_scc_manager.po (hunk 2 above)
Comment (nadjaheitmann) [reply locale/en/hammer_cli_foreman_scc_manager.po:8]: I checked in other projects and it does not seem to be in line with the actual plugin version... So let's just ignore it for now.

**Review by nadjaheitmann — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Thanks Oleh!

**Pull request merged**